src/schema/v2/partner/__tests__/partnerCollectorProfile.ts

```diff
@@ -0,0 +1,32 @@
+import { GraphQLObjectType } from "graphql"
+import { CollectorProfileType } from "schema/v2/CollectorProfile/collectorProfile"
+import { ResolverContext } from "types/graphql"
+
+export const PartnerCollectorProfileType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "PartnerCollectorProfile",
+  fields: () => ({
+    collectorProfile: {
+      type: CollectorProfileType,
+      resolve: async (
+        { partnerId, collectorId },
+        _args,
+        { partnerCollectorProfileLoader }
+      ) => {
+        if (!partnerCollectorProfileLoader) return
+
+        const data = await partnerCollectorProfileLoader({
```

Comment:
**suggestion:** This shouldn't be part of the `__tests__` directory? 
Also could we make this a bit more defensive? For instance, adding a `try catch` block.